karate-core/src/main/java/com/intuit/karate/FileUtils.java

```diff
@@ -479,6 +479,19 @@ private static Path getPathFor(URL url, String relativePath) {
                     URL url = new URL("jar:" + u + "!/");
                     list.add(url);
                 }
+            } else {
+                String classpath = System.getProperty("java.class.path");
```

Comment:
was trying to read up on the difference between `System.getProperty("java.class.path")` and `getClassLoader.getURLs()` - refer https://stackoverflow.com/a/18627625/143475

wondering if we should just stick to one way of doing so.

Reply:
getClassLoader().getURLs() is the same as in the if block above - i.e. it relies on getClassLoader returning a URLClassLoader, which in JDK9+, it doesn't (note the ages on that stackoverflow q/a).

I hunted around for other ways and couldn't see much - happy to use anything better if you find something though.

Reply:
no worries, we can always pivot this later. was just wondering if we can get rid of the `getURLs()` altogether

Reply:
oh sorry i misunderstood. I think the "java.class.path" method would solve even the JDK8 and below cases, I was just gun-shy of changing existing code...